### PR TITLE
Use vanilla slurm appliance for CaaS slurm

### PR DESCRIPTION
**Pull request by sjpb**

### CODE REVIEW

**Review by JohnGarbutt — APPROVED** [review]
This is looking good to me (although I do want to see the functional test pass)

File: roles/azimuth_caas_operator/defaults/main.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # The chart to use
 azimuth_caas_operator_chart_repo: https://stackhpc.github.io/azimuth-caas-operator
 azimuth_caas_operator_chart_name: azimuth-caas-operator
-azimuth_caas_operator_chart_version: 0.5.0
+azimuth_caas_operator_chart_version: '0.6.0'
```
Comment (sd109) [inline roles/azimuth_caas_operator/defaults/main.yml:5]: Nit:
```suggestion
azimuth_caas_operator_chart_version: 0.6.0
```

**Review by sd109 — APPROVED** [review]
LGTM assuming tests pass

File: roles/azimuth_caas_operator/defaults/main.yml (hunk 1 above)
Comment (sd109) [reply roles/azimuth_caas_operator/defaults/main.yml:5]: Nvm, turns out this is safer and all other non-quoted instances in this file should be fixed instead...

**Pull request merged**